src/components/NewLexemeForm.vue

```diff
@@ -39,6 +39,15 @@ const lexicalCategory = computed( {
 	},
 } );
 const token = computed( () => store.state.token );
+const submitMsg = $messages.get( 'wikibaselexeme-newlexeme-submit' );
+const termsOfUseTitle = $messages.get( 'copyrightpage' );
+const copyrightText = $messages.get(
+	'wikibase-shortcopyrightwarning',
+	submitMsg,
+	termsOfUseTitle,
+	store.state.config.licenseUrl,
+	store.state.config.licenseName,
```

Comment:
I wonder if we should make this message `computed()`, because the store state could theoretically change? (We don’t expect it to actually change, but the same is true of the token, immediately above, and that is still computed.)

Reply:
Looks like the rule is [member-delimiter-style](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/member-delimiter-style.md), and it’s not part of our config at the moment. I’ll open a separate PR to propose adding it.

Reply:
wmde/eslint-config-wikimedia-typescript#15

Reply:
I don't think this would ever change, we are inheriting this from outside the app, and only using the store to avoid drilling down properties at this point, but this raises a good point that I think we discussed a few weeks ago which is to create a specialized plugin just for the config something that we decided to do down the line. Please feel free to create a task for that on the backlog, though

Reply:
Created T303050. Let’s leave this as it is in the meantime.